src/Microsoft.AspNet.Authentication.Cookies/CookieAuthenticationHandler.cs

```diff
@@ -279,6 +279,7 @@ private CookieOptions BuildCookieOptions()
             var context = new CookieSigningOutContext(
                 Context,
                 Options,
+                new AuthenticationProperties(signOutContext.Properties),
```

Comment:
Dev style mostly, I prefer ctor.  I think @Tratcher preferred properties.  I can make them consistent and ctor required everywhere I guess.  Chris thoughts?